javascript.html.markdown

```diff
@@ -606,17 +606,15 @@ pi = 4.13; // You cannot do this.
 // There is a new syntax for functions in ES6 known as "lambda syntax".
 // This allows functions to be defined in a lexical scope like with variables
 // defined by const and let. 
+// By default, a single statement defined will be the return statement, hence the `return` keyword is not required. 
 
-const isEven = (number) => {
-    return number % 2 === 0;
-};
-
-isEven(7); // false
+const isOdd = number => number % 2 !== 0;
 
+// Parentheses are not required if only one argument is specified;
```

Comment:
however, they are required when there are no arguments
i feel we should demonstrate both
something like a simple
```
const sayHello = () => {
     console.log("Hello World") 
   }
```

Reply:
Added in the following commit https://github.com/adambard/learnxinyminutes-docs/pull/3842/commits/9d11712b8bc5207763dd6546c4551fb76125f0a6